static/src/javascripts/projects/commercial/modules/close-disabled-slots.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import qwery from 'qwery';
+import fastdom from 'lib/fastdom-promise';
+
+const adSlotSelector: string = '.js-ad-slot';
+
+const shouldDisableAdSlot = adSlot =>
+    window.getComputedStyle(adSlot).display === 'none';
+
+const closeDisabledSlots = (force: boolean): Promise<void> => {
```

Comment:
@regiskuckaertz It might have something to do with flow not covering the tests? In actual code we seem to always pass the boolean.